app/controllers/support/users_controller.rb

```diff
@@ -21,6 +21,14 @@ def create
       end
     end
 
+    def destroy
+      if user.destroy
```

Comment:
Does `destroy` call the `discard` action? i.e. soft-delete

Reply:
No, a hard delete; changed to `discard` for soft-delete 👍